src/Crud/Controller/Get.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace Biig\Melodiia\Crud\Controller;
+
+use Biig\Melodiia\Crud\CrudControllerInterface;
+use Biig\Melodiia\Crud\Persistence\DataStoreInterface;
+use Biig\Melodiia\Response\OkContent;
+use Symfony\Component\HttpFoundation\Request;
+
+final class Get implements CrudControllerInterface
+{
+    /** @var DataStoreInterface */
+    private $dataStore;
+
+    public function __construct(DataStoreInterface $dataStore)
+    {
+        $this->dataStore = $dataStore;
+    }
+
+    public function __invoke(Request $request, $id)
+    {
+        $modelClass = $request->attributes->get(self::MODEL_ATTRIBUTE);
+        $groups = $request->attributes->get(self::SERIALIZATION_GROUP, []);
+
+        return new OkContent($this->dataStore->find($modelClass, $id), $groups);
```

Comment:
if `$id` don't exist, IMO, you should return a 404 response

Reply:
Fixed!